ex3/app/src/main/java/com/example/myapplication/Notification_receiver.java

```diff
@@ -0,0 +1,65 @@
+package com.example.myapplication;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+import androidx.core.app.NotificationCompat;
+
+import java.util.Random;
+
+public class Notification_receiver extends BroadcastReceiver {
+    @RequiresApi(api = Build.VERSION_CODES.O)
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String[] quotesArray = new String[] {
+                "We were on a break!",
+                "Well, maybe I don’t need your money. Wait, wait, I said maybe!",
+                "Joey doesn’t share food!",
+                "Hi, I’m Chandler. I make jokes when I’m uncomfortable.",
+                "I wish I could, but I don’t want to",
+                "Pivot!",
+                "It tastes like feet!",
+                "What’s not to like? Custard: good. Jam: good. Meat: good!",
+                "Your little Harmonica is hammered.",
+                "How you doin'?",
+                "Oh. My. God.",
+                "I don’t even have a ‘pla.'"
+        };
+        Random rand = new Random();
+        int randLocation = rand.nextInt(15); //Generating a random number to chose a quote from the array
```

Comment:
Use quotesArray.length instead of hard coding the number of elements in the array